.github/workflows/node.js.yml

```diff
@@ -17,20 +17,17 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x, 15.x]
```

Comment:
I am dropping version 10 and 15 which is not supported anymore https://nodejs.org/en/about/releases/

Reply:
it allows us to upgrade to npm 8